Apps/WebClient/src/ClientApp/public/index.js

```diff
@@ -0,0 +1,16 @@
+function detectBrowser() {
+    const body = document.getElementsByTagName("BODY")[0];
+    const unsupportedBrowser = document.getElementById("unsupported-browser");
+    console.log("Modernizr.es6number: " + Modernizr.es6number);
+    if (Modernizr.es6number) {
+        body.removeChild(unsupportedBrowser);
+    } else {
+        body.removeChild(document.getElementById("app-root"));
+        const unsupportedBrowserMsg =
+            "Health Gateway is not compatible with this browser. Please use another browser.";
+        unsupportedBrowser.innerHTML = unsupportedBrowserMsg;
+    }
+}
+window.onload = function () {
+    detectBrowser();
```

Comment:
moving this code from <body..> of the index.html to this js code has helped to prevent the "Refused to execute inline event handler...violates the following Content Security Policy..." yeah!
![image](https://user-images.githubusercontent.com/48332333/93535615-28e89680-f8fc-11ea-8b8d-e26e8c596012.png)